### PR TITLE
Update committee Section on Website

### PR DESCRIPTION
**Pull request by erinosher**

### CODE REVIEW

**Review by AksanDotDev — APPROVED** [review]
Adjusted my socials, else it's looking good.

File: src/components/pages/HomePage/HomePage.jsx [hunk 1]
```diff
@@ -276,136 +277,105 @@ const HomePage = () => (
...
+              {/* <a href='https://twitter.com' target='_blank' rel='noopener noreferrer'>
                 <Icon link name='twitter' className={styles.twitter} />
-              </a>
-              <a href='https://linkedin.com/in/viktoriapopovici' target='_blank' rel='noopener noreferrer'>
+              </a> */}
+              {/* <a href='https://www.linkedin.com/in/' target='_blank' rel='noopener noreferrer'>
                 <Icon link name='linkedin' className={styles.linkedIn} />
-              </a>
+              </a> */}
             </p>
```
Comment (Fraetor) [inline src/components/pages/HomePage/HomePage.jsx:359]:
```suggestion
            <a href="https://www.frost.cx">
              <Icon link name="globe" color="green">
            </a>
            </p>
```

Comment (Fraetor) [reply src/components/pages/HomePage/HomePage.jsx:359]: I'm guessing with the icon name, so that may need fixing.

**Review by Fraetor — APPROVED** [review]
I've added a link to my website, bit it otherwise looks good.

File: src/components/pages/HomePage/HomePage.jsx (hunk 1 above)
Comment (erinosher) [reply src/components/pages/HomePage/HomePage.jsx:359]: Just added this on #155

**Pull request merged**